test/smoke_test/bcda_client.go

```diff
@@ -307,7 +309,10 @@ func (c *client) updateAccessToken() error {
 	req.SetBasicAuth(clientID, clientSecret)
 	req.Header.Add("Accept", "application/json")
 
-	resp, err := c.httpClient.Do(req)
+	// The retry logic may try to update the access token, to avoid a recursive
+	// retry loop we dont want to use the retry logic on updating the access token
+	http.DefaultClient.Timeout = httpTimeout
```

Comment:
Does it make sense to put this on line 52?

Reply:
That would be fine. I kept it where it is since its the only place we currently reference the http client... but if we move it up it'll only set it once.